lib/tableau_dev_server/router.ex

```diff
@@ -32,11 +33,21 @@ defmodule TableauDevServer.Router do
   end
 
   defp recompile(conn, _) do
-    if conn.request_path != "/ws" do
-      WebDevUtils.CodeReloader.reload()
+    if conn.request_path == "/ws" do
+      conn
+    else
+      case WebDevUtils.CodeReloader.reload() do
+        {:error, errors} ->
+          errors = Enum.filter(errors, &(&1.severity == :error))
+          message = Enum.reduce(errors, "", fn error, acc -> "#{acc}\n#{error.message}" end)
```

Comment:
I think you just want Enum.map_join here

`Enum.map_join(errors, "\n", & &1.message)`

Reply:
Ok, I updated the code